app/models/financialDetails/MfaDebitUtils.scala

```diff
@@ -17,14 +17,14 @@
 package models.financialDetails
 
 object MfaDebitUtils {
-  private val MFADebitType: Map[Int, String] = Map(
-    4004 -> "ITSA PAYE Charge",
-    4005 -> "ITSA Calc Error Correction",
-    4006 -> "ITSA Manual Penalty Pre CY-4",
-    4007 -> "ITSA Misc Charge")
+  private val MFADebitType: List[String] = List(
```

Comment:
If we can use "Main transaction code" instead to detect MFA charges:
> codes between 4000-4003 see below
![Uploading image.png…]()